extensions/dacpac/src/test/testContext.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+export interface TestContext {
+	context: vscode.ExtensionContext;
+}
+
+export function createContext(): TestContext {
+	let extensionPath = path.join(__dirname, '..', '..');
+
+	return {
+		context: {
+			subscriptions: [],
+			workspaceState: {
+				get: () => { return Promise.resolve(); },
+				update: () => { return Promise.resolve(); }
+			},
+			globalState: {
+				get: () => { return Promise.resolve(); },
+				update: () => { return Promise.resolve(); }
+			},
+			extensionPath: extensionPath,
+			asAbsolutePath: () => { return ''; },
+			storagePath: '',
+			globalStoragePath: '',
+			logPath: '',
+			extensionUri: vscode.Uri.parse(''),
+			environmentVariableCollection: undefined as any
```

Comment:
evnironmentVariableConnection got added as required in ExtensionContext in master last night between the time I last updated my PR and merged it this morning, which caused the build break.